packages/web/components/Guild/GuildTile.tsx

```diff
@@ -46,14 +51,6 @@ export const GuildTile: React.FC<Props> = ({ guild }) => (
       ) : null}
     </MetaTileHeader>
     <MetaTileBody>
-      <VStack spacing={2} align="stretch">
-        <Text fontFamily="mono" fontSize="sm" color="blueLight">
-          LINKS
-        </Text>
-        <HStack mt="2">
-          <GuildLinks guild={guild} />
-        </HStack>
-      </VStack>
```

Comment:
What is the rationale for removing the links ?

Reply:
Cause the links are in a different place on the guild details page

Reply:
Seems useful to have them in the tiles at-a-glance as well ? There isn't a whole lot of info at the moment on a particular tile

Reply:
Yeah well we could keep them in the GuildHero than we need to update the design.
A guild could have guite a lot of links tho which makes the hero a bit too messy